### PR TITLE
PS-4680 PS & PXC - Document cnf file locations and best practices

### PR DESCRIPTION
**Pull request by patrickbirch**
modified file:
source/installation/apt_repo
source/installation/yum_repo

### CODE REVIEW

File: doc/source/installation/apt_repo.rst [hunk 1]
```diff
@@ -69,13 +69,13 @@ Installing |Percona Server| from Percona ``apt`` repository
...
 
-      $ sudo apt-get install percona-server-server-5.6 
+      $ sudo apt-get install percona-server-server-5.6
 
 
 Percona ``apt`` Testing repository
 ----------------------------------
 
-Percona offers pre-release builds from the testing repository. To enable it add the just uncomment the testing repository lines in the Percona repository definition in your repository file (default :file:`/etc/apt/sources.list.d/percona-release.list`). It should looks like this (in this example ``VERSION`` is the name of your distribution): :: 
+Percona offers pre-release builds from the testing repository. To enable it add the just uncomment the testing repository lines in the Percona repository definition in your repository file (default :file:`/etc/apt/sources.list.d/percona-release.list`). It should looks like this (in this example ``VERSION`` is the name of your distribution): ::
```
Comment (hrvojem) [inline doc/source/installation/apt_repo.rst:86]: New recommendation is to use `percona-release` command instead of manually editing the configs

**Review by tplavcic — APPROVED** [review]
LGTM

**Review by hrvojem — APPROVED** [review]
LGTM

File: doc/source/installation/apt_repo.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/installation/apt_repo.rst:86]: Added. 
the change also added several .res files and a change to the conf.py.

**Pull request merged**